frontend/micro-ui/web/micro-ui-internals/packages/css/src/home/home.scss

```
@@ -670,6 +670,13 @@
   outline: none;
 }
 
+.manage-office-search-field__country:disabled {
+  background-color: #f5f5f5;
+  color: #666666;
```

Stylelint (17.2.0)

[error] Expected "#666666" to be "#666" (color-hex-length)

(color-hex-length)